packages/xrpl/HISTORY.md

```diff
@@ -5,7 +5,11 @@ Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xr
 ## Unreleased Changes
 
 ### Added
-* parseTransactionFlags as a utility function in the xrpl package to streamline transactions flags-to-map conversion
+* `parseTransactionFlags` as a utility function in the xrpl package to streamline transactions flags-to-map conversion
+* Support for XLS-70d Verifiable Credentials
+  * New transactions CredentialCreate, CredentialAccept, CredentialDelete
```

Comment:
I feel lines 10-12 can be removed because XLS-70d conveys enough information, but I don't feel too strongly about it

Reply:
I don't feel strongly about it either, just felt its nice to have in case someone didn't fully read the spec they still get a high level of what new transactions were added

Reply:
sure, made the change